2025-02-17 16:53:50 - ERROR - 加載配置文件時出錯: [Errno 2] No such file or directory: 'archive/20240720/config.yaml'
2025-02-17 16:53:50 - ERROR - 程序執行過程中發生錯誤: [Errno 2] No such file or directory: 'archive/20240720/config.yaml'
Traceback (most recent call last):
  File "d:\marskingx.github.io\reindex\main.py", line 105, in main
    config = load_config()
             ^^^^^^^^^^^^^
  File "d:\marskingx.github.io\reindex\main.py", line 29, in load_config
    with open('archive/20240720/config.yaml', 'r', encoding='utf-8') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'archive/20240720/config.yaml'
2025-02-17 16:54:02 - ERROR - 加載配置文件時出錯: [Errno 2] No such file or directory: 'archive/20240720/config.yaml'
2025-02-17 16:54:02 - ERROR - 程序執行過程中發生錯誤: [Errno 2] No such file or directory: 'archive/20240720/config.yaml'
Traceback (most recent call last):
  File "d:\marskingx.github.io\reindex\main.py", line 105, in main
    config = load_config()
             ^^^^^^^^^^^^^
  File "d:\marskingx.github.io\reindex\main.py", line 29, in load_config
    with open('archive/20240720/config.yaml', 'r', encoding='utf-8') as file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'archive/20240720/config.yaml'
2025-02-17 16:54:34 - ERROR - 程序執行過程中發生錯誤: 'NoneType' object is not subscriptable
Traceback (most recent call last):
  File "d:\marskingx.github.io\reindex\main.py", line 100, in main
    directory_path = config['file_paths']['blog_directory']
                     ~~~~~~^^^^^^^^^^^^^^
TypeError: 'NoneType' object is not subscriptable
